heracal/tests/test_omni.py

```diff
@@ -169,7 +205,6 @@ def test_from_fits_delay(self):
 
         nt.assert_equal(xtalk, {})
 
-
```

Comment:
I.e., getting rid of an extra blank line is not what this PR is about, so it's better not to include the change here.